### ISSUE TITLE
Inconsistent button-behavior when hover

### ISSUE THREAD

**Issue by NusiNusi** [Ubuntu/gtk-communitheme#164]
When you hover a button it changes it color to show the user that he is near to an action.
The bright grey-buttons react completely different compared to the dark grey-buttons. The brighter buttons get darker when hovered and the darker buttons get brighter. For consistency both should get brighter or at least behave the same.
![greybutton](https://user-images.githubusercontent.com/36449926/36162817-cd646f90-10e7-11e8-9056-38cfa7b6849c.png)
This is how the grey button looks.

![darkerhover](https://user-images.githubusercontent.com/36449926/36162901-02ce72c0-10e8-11e8-8beb-2255e9159fa5.png)
...and this when hovered. Maybe it should get brighter instead.

The same goes for green and red buttons.
![green](https://user-images.githubusercontent.com/36449926/36162951-2a60ab3c-10e8-11e8-86e4-fbc275e0ed9f.png)

![greenhover](https://user-images.githubusercontent.com/36449926/36162971-35fb3e58-10e8-11e8-9d54-3ebb1b38fafb.png)

The dark buttons get brighter (grey).
![dark](https://user-images.githubusercontent.com/36449926/36163003-53631196-10e8-11e8-812a-06e8a6f75c8c.png)

![darkhover](https://user-images.githubusercontent.com/36449926/36163019-5c23a66a-10e8-11e8-9505-f76ef3ae055d.png)

Why should buttons get brighter when hovered? Brightness implicates you are closer to something. When you hover a button you are closer to the action. Button has "insert color" -> hovering it -> you are closer to the action -> this needs your attention, you are going to do something -> "insert color"-button gets brighter.

**Comment by clobrano** [comment]
Same behavior on Ambiance, but let's see what the designers think about it :wink: 

![hover-color](https://user-images.githubusercontent.com/2883614/36164235-12080b8a-10ec-11e8-83db-30f914ee83ee.gif)

(and on the Github green "Comment" button below :joy: )

**Comment by NusiNusi** [comment]
Startingpoint is not ambiance. Adwaita is:
https://community.ubuntu.com/t/call-for-participation-an-ubuntu-default-theme-lead-by-the-community/1545?u=nusi
At least this is what @didrocks said.
In adwaita buttons get brighter (at least for me this seems logic):
![adwaita](https://user-images.githubusercontent.com/36449926/36166003-926d7594-10f1-11e8-824f-a5db3753264c.png)
![adwaitahover](https://user-images.githubusercontent.com/36449926/36166006-9663c4fa-10f1-11e8-88fa-7fd80417a111.png)

But you are right, lets see what the designers think about it.

**Comment by Luxamman** [comment]
Well, it is more consistency than adding a shadow IMHO^^
Also, we already have decided to not use shadows for now because of several problems in some circumstances. That is why the buttons have borders, not shadows. And sometimes no background and borders (also kind of inconsistency?). 

The only way I see is to reverse it and make all of them brighter on hover - this way, we approach the dark buttons. But can lead to worse readability because of the white text. 

We are including the Suru design (buttons get darker on hover), combining it with a dark flat design for the header - and the flat dark button needs a bright hover, course. 

The questions are: Is it irritating or misleading? Is it not fulfilling the purpose of it? Is it really destroying the logic of it, or will no one pay attention? 

If the programmers want to try it to see how it look they can maybe give feedback. Else, I would suggest giving it attention for the next tests in the live system for a final statement.

**Comment by NusiNusi** [comment]
To your questions:
Is it irritating or misleading? - No. We still know what is meant.
Is it not fulfilling the purpose of it? - No. Still makes clear that we hover a button.
Is it really destroying the logic of it, or will no one pay attention? - Yes. It is destroying the logic. I explained this in the first post. It looks like the theme is put together by two different themes with different design-languages. I think most people wont pay attention but will "feel" that something is anomalous about the look of the OS they are using.

**Comment by clobrano** [comment]
> Startingpoint is not ambiance. Adwaita is

Adwaita doesn't have dark buttons though

**Comment by NusiNusi** [comment]
Adwaita Dark has! And they get brighter too.

**Comment by Luxamman** [comment]
@madsrh yes, since also overall discussions about buttons are going on, we can close and overlook in a few weeks when changes are implemented - also we keep that issue in mind, course!
